Update Bits AI SRE Docs (#30193)
DataDog/documentation: content/en/bits_ai/bits_ai_sre/investigate_alerts.md

- **Option 2: Add the Bits AI tag to a single monitor**
  1. Open a monitor, click on the gear icon in the upper right corner, and select **Edit**.
  1. Add the `bitsai:enabled` tag. 
  1. Save your changes.
- **Option 3: Add the Bits AI tag in bulk**
  1. In the **Monitor List**, select multiple monitors, then click **Edit tags**.
  1. Add the `bitsai:enabled` tag to the selected monitors.

You can also add the tag to your desired monitors using the Datadog API or Terraform. 

An investigation initiates when a monitor transitions to the alert state. Transitions to the warn or no data state, [renotifications][12], and test notifications do not trigger investigations. Additionally, noisy monitors are automatically rate limited to avoid unnecessary investigations and protect your budget.

### Manually start an investigation

Alternatively, you can manually invoke Bits on an individual monitor event. 

- **Option 1: Monitor Status Page**
  -  On the monitor status page, select an alert event, then click **Investigate with Bits AI**.
- **Option 2: Monitor Event Side Panel**
  -  On the monitor event side panel, click **Investigate with Bits AI**.
- **Option 3: Slack**

>   - Under a monitor notification in Slack, mention `@Datadog Investigate this alert`.

Reviewer: ```suggestion
  - In Slack, reply to a monitor notification with `@Datadog Investigate this alert`.
```

In the **Configure notifications and automations** section, add the `@case-{project-name}` handle.
{{% /collapse-content %}}

## How Bits AI SRE investigates

Investigations happen in two phases:

1. **Initial context gathering**

>    1. Bits begins by identifying any Datadog links that you've added to the monitor's message and uses them as entry points into the investigation.

Reviewer: ```suggestion
   1. Bits begins by identifying any Datadog links that you have added to the monitor's message and uses them as entry points into the investigation.
```
